static/js/message_edit.js

```diff
@@ -355,7 +355,7 @@ function edit_message(row, raw_content) {
             stream_name: message.stream,
             notify_new_thread: notify_new_thread_default,
             notify_old_thread: notify_old_thread_default,
-            giphy_api_available: page_params.giphy_api_key !== "",
+            giphy_enabled: page_params.giphy_api_key !== "",
```

Comment:
I assume this last commit is going to have the other Giphy stuff rebased on top to change this check.  Looks great.

Reply:
Yeah, there are other css related changes too that I have to make.